src/Files.App/MainWindow.xaml.cs

```diff
@@ -181,6 +181,10 @@ public async Task InitializeApplicationAsync(object activatedEventArgs)
 						await mainPageViewModel.AddNewTabByPathAsync(typeof(PaneHolderPage), fileArgs.Files[index].Path);
 					}
 					break;
+
+				case IStartupTaskActivatedEventArgs startupArgs:
```

Comment:
Would it be better to use `default` instead of specifying the interface?

Reply:
We could use both, this way it's clear that they have the same behavior.

Reply:
This might also be useful if/when we add an option to open the app in the background.